resources/js/pages/ProfessionalFieldDecision.vue

```diff
@@ -0,0 +1,391 @@
+                <p class="text-2xl text-center mb-4">Deine Auswahl:</p>
+                <div
+                    class="p-4 m-2 min-w-full text-xl flex border border-gray-200 rounded-lg shadow max-h-fit items-center justify-between">
+                    <div></div>
+                    <p>{{ selectedPresentation.name }}</p>
+                    <button class="ml-2" :value="selectedPresentation" @click="showModal(selectedPresentation)">
+                        <font-awesome-icon icon="fa-solid fa-circle-info" class="text-bit-blue"/>
+                    </button>
+                </div>
+
+                <div v-for="field in selectedFields.values()" :key="field.id"
+                     class="p-4 m-2 min-w-full text-xl flex border border-gray-200 rounded-lg shadow max-h-fit items-center justify-between">
+                    <div></div>
+                    <p>{{ field.name }}</p>
+                    <button class="ml-2" :value="field" @click="showModal(field)">
+                        <font-awesome-icon icon="fa-solid fa-circle-info" class="text-bit-blue"/>
+                    </button>
+                </div>
+            </template>
+        </Wizard>
+    </Layout>
+
+    <BitModal v-model="show" :title="modalTitle" :text="modalText" @confirm="hideModal"></BitModal>
+
+</template>
+
+<script setup>
+import Wizard from 'form-wizard-vue3'
+import 'form-wizard-vue3/dist/form-wizard-vue3.css'
+import {FontAwesomeIcon} from "@fortawesome/vue-fontawesome";
+import Layout from "@/components/Layout.vue";
+import {ref} from "vue";
+import BitModal from "@/components/BitModal.vue";
+
+const props = defineProps({
+    professional_fields: Array,
+    general_presentations: Array,
+});
+
+
+let currentTabIndex = ref(0);
+
+let selectedPresentation = ref(null);
+
+let selectedFields = ref([]);
+
+
+const nextButton = ref({
+    text: 'Weiter',
+    hideIcon: 'true',
+    disabled: true
+});
+
+function enableNextButton() {
+    nextButton.value.disabled = false;
+}
+
+function disableNextButton() {
+    nextButton.value.disabled = true;
+}
+
+function validate() {
+    if (currentTabIndex.value === 0 || currentTabIndex === 0) {
+        validatePresentation();
+    } else if (currentTabIndex.value === 1 || currentTabIndex === 1) {
+        validateFields();
+    }
+}
+
+
+function validatePresentation() {
+    if (selectedPresentation.value === null) {
+        disableNextButton();
+    } else {
+        enableNextButton();
+    }
+}
+
+function validateFields() {
+    if (!(selectedFields.value.length === 2)) {
+        disableNextButton();
+    } else {
+        enableNextButton();
+    }
+}
+
+function isAvailable(field) {
+    return (field.current_count < field.max_count);
+}
+
+function onChangeCurrentTab(index) {
+    currentTabIndex = index;
+    validate();
+}
+
+function wizardCompleted() {
+    //TODO: Save the data to the database @Chris-Backend
+    //I'am not sure how to do it to satisfy your requirements
```

Comment:
I can implement that, if you show me, how to send data with vue and inertia

Reply:
We just have to pass a the professionalFields and the presentation with a inertia route as arguments

Reply:
I wasn't sure how to deal with the update or store route thing